### PR TITLE
ct-1849 add message to overturned sar

### PR DESCRIPTION
**Pull request by beccapearce**

### CODE REVIEW

File: spec/features/cases/overturned_sar/add_message_to_case_spec.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# This would be better placed in e2e journey spec
+
+require 'rails_helper'
+
+feature 'SAR Case creation by a manager' do
```
Comment (misaka) [inline spec/features/cases/overturned_sar/add_message_to_case_spec.rb]: Old feature text in here.

**Review by misaka — CHANGES_REQUESTED** [review]
I think the test should either be part of the end-to-end journey or rely on a factory.

**Review by misaka — APPROVED** [review]

**Comment by misaka** [comment]
Do not merge until we get CT-1914 done.

**Comment by aliuk2012** [comment]
#422 has been merged

**Pull request merged**